src/font-faces.css

```diff
@@ -9,68 +9,274 @@
  * reduces wait time for fonts that we know we are using.
  */
 
-/*
+/* ––––––––––––––––––––––––––––––––––––––––––––––––––––––––––––––
+Font: Circular
+Cuts: Thin, Thin Italic, Light, Light Italic, Regular, Italic, Book, Book Italic, Medium, Medium Italic, Bold, Bold Italic, Black, Black Italic, ExtraBlack, ExtraBlack Italic
+Format: woff, woff2
+
+Purchase Date: November 5, 2019
+––––––––––––––––––––––––––––––––––––––––––––––––––––––––––––––
+
+Legal Disclaimer for Web licence
+
+Licence to use self-hosted webfonts for displaying dynamic text on specified website domains. Our package includes WOFF and WOFF2 font formats.
+
+Should the license page views limit be exceeded the license holder will be required to contact Lineto.com for an upgrade.
+
+It is strictly prohibited to rename the font and to download or use these fonts in any other media.
+
+These Web fonts are licensed exclusively for the use on the following domain(s) and their subdomains:
+
+sharetribe.com (Monthly Pageviews: <250K)
+
+––––––––––––––––––––––––––––––––––––––––––––––––––––––––––––––
+
+INSTRUCTIONS
+
+The Web Font delivery folder contains all purchased Web Fonts, example CSS
+file containing @font-face feature statements, and HTML Sample Pages.
+You are only allowed to use the cuts listed above for the licensed
+application.
+
+We recommend using the @font-face statements from the example CSS file.
+Depending on your application structure, you may need to adjust the
+paths to the fonts. It is prohibited to change font/file names.
+
+Please see the HTML Sample Page in the font delivery package for further
+information about Character Groups, Language Support and
+OpenType features etc.
+
+––––––––––––––––––––––––––––––––––––––––––––––––––––––––––––––
+
+Manufacturer  Lineto GmbH
+
+Vendor URL    https://lineto.com
+
+Copyright     Copyright © 2019 by Lineto GmbH.
+              https://lineto.com
+              All rights reserved
+
+Designer
+              https://lineto.com/designers
+
+EULA          https://lineto.com/assets/pdf/EULA.pdf
+
+*/
+
+
 @font-face {
-  font-family: CircularStd;
-  src: url("./font-files/circular-std/CircularStd-Black.woff2") format("woff2");
-  font-weight: 800;
+  font-family: "Circular";
+  src: url("https://assets-sharetribecom.sharetribe.com/webfonts/circular/CircularXXSub-ThinSubset.woff2") format("woff2");
+  font-weight: 100;
   font-style: normal;
-  font-display: fallback;
 }
 
 @font-face {
-  font-family: CircularStd;
-  src: url("./font-files/circular-std/CircularStd-BlackItalic.woff2") format("woff2");
-  font-weight: 800;
+  font-family: "Circular";
+  src: url("https://assets-sharetribecom.sharetribe.com/webfonts/circular/CircularXXSub-ThinItalicSubset.woff") format("woff");
```

Comment:
Why do you add both the `woff2` and `woff` formats if only the first was enough previously? Is there some browser support that we were missing before? How does that affect what files browsers download?

Reply:
"Web Open Font Format 2 (WOFF2): WOFF2 is an update to the original WOFF format. Developed by Google, this is considered the best format of the bunch because it offers smaller file sizes and better performance for modern browsers that support it." -  https://css-tricks.com/understanding-web-fonts-getting/

So WOFF2 is an update to WOFF, but having WOFF as a backup to older browsers is still recommended.

Reply:
We can of course add support for older browsers, but WOFF2 support looks pretty good already: https://caniuse.com/?search=woff2

I'd personally go just with WOFF2 here as the support is good and the fallback behavior doesn't break anything. It simplifies the code, but feel free to decide otherwise if you think the browser support isn't enough.

Reply:
I would like to keep the WOFF still in. All major font providers still include WOFF and WOFF2 files in their code and deliverables and WOFF2's global rate is ~95% and WOFF's is ~97.5%. Supporting both doesn't increase page load, or render times and the font-faces.css file is not something that we (hopefully) have to touch regularly.

To me, the pro (wider support) outweights the con (more code in font-faces.css files) and I would like to keep them both in. 

But if you'd like to simplify the code without sacrificing the support, we could remove the unused styles and weights from the code :)